### PR TITLE
Book library app

### PR DESCRIPTION
**Pull request by tylher**
- Create a collection for storing books
- Implement add and Remove buttons to add and remove books respectively

### CODE REVIEW

File: awesome_book.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  Book = Book.filter((book) => book.Title !== e.parentElement.childNodes[0].textContent);
+
+  /* eslint-disable */
+    Book.map((book) => {
+        /* eslint-enable */
+    const div = document.createElement('div');
+    div.innerHTML = `<h2>${book.Title}</h2> <p>${book.author}</p> <button onclick='remove(this)' class='remove-book'>remove</button> <hr>`;
+    displayBooks.appendChild(div);
+  });
+}
```
Comment (thecodechaser) [inline awesome_book.js:51]: - [ ] Good job on the functionalities but there is an issue. Remove book functionality isn't working. Please make sure removing books is working fine and users are able to remove the book from the collections after adding it.

### See the image for reference 

image

File: awesome_book.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+const formElements = document.querySelectorAll('input, textarea');
+formElements.forEach((element) => {
+  element.value = formData[element.name];
+  element.addEventListener('input', (e) => {
+    formData[e.target.name] = e.target.value;
+    localStorage.setItem('formData', JSON.stringify(formData));
+  });
+});
```
Comment (thecodechaser) [inline awesome_book.js]: - [ ] You are supposed to save and fetch book data to local storage, not the form data. Please make sure after adding a book to the collection the book is saved to local storage and if we refresh the page the added books are fetched from local storage and visible on-screen(collection).

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @tylher and @Shubh-Dev 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .vscode/settings.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
+  "liveServer.settings.port": 5501,
+  "editor.defaultFormatter": "stylelint.vscode-stylelint",
+  "[html]": {
+    "editor.defaultFormatter": "vscode.html-language-features"
+  },
+  "[javascript]": {
+    "editor.defaultFormatter": "dbaeumer.vscode-eslint"
```
Comment (Cathella) [inline .vscode/settings.json:8]: - [x] [OPTIONAL] Please kindly add this file to the `.gitignore` file because it is not directly related to the project you are working on 🙏🏻.

**Review by Cathella — APPROVED** [review]
## 🔥 Status: Approved 🟢

### Hi @tylher, @Shubh-Dev 👋🏻, 

Good job on making the requested changes 💪🏻!
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💪🏻

✅ User can delete book


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .vscode/settings.json (hunk 3 above)
Comment (tylher) [reply .vscode/settings.json:8]: Noted. Thank you!
